cpp/models/ode_secirvvs/parameters_io.h

```diff
@@ -753,636 +835,82 @@ IOResult<void> set_vaccination_data(std::vector<Model<FP>>& model, const std::st
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedSymptomsPartialImmunity +
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedSymptomsImprovedImmunity +
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedSevereNaive + age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedSeverePartialImmunity +
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedSevereImprovedImmunity +
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedCriticalNaive +
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedCriticalPartialImmunity +
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::InfectedCriticalImprovedImmunity +
-                                                       age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::DeadNaive + age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::DeadPartialImmunity + age_group_offset) +
-                         extrapolated_rki[county][day]((size_t)InfectionState::DeadImprovedImmunity +
-                                                       age_group_offset));
-
-                    // in set_confirmed_cases_data initilization, deaths are now set to 0. In order to visualize
-                    // the extrapolated real number of deaths, they have to be set here. In the comparison of data
-                    // it has to be paid attention to the fact, the the simulation starts with deaths=0
-                    // while this method starts with deaths=number of reported deaths so far...
-                    // Additionally, we set the number of reported deaths to DeadNaive since no information on that is
-                    // available here.
-                    // Do only add deaths after substraction.
-                    extrapolated_rki[county][day]((size_t)InfectionState::DeadNaive + age_group_offset) =
-                        num_death_uv[county][age];
-                }
-            }
-            else {
-                log_warning("No population data available for region " + std::to_string(county) +
-                            ". Population data has not been set.");
+        BOOST_OUTCOME_TRY(
```

Comment:
as in the other file: use the functions that uses paths to avoid code duplication

Reply:
see above